graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/DgsDataFetchingEnvironment.kt

```diff
@@ -51,4 +52,30 @@ class DgsDataFetchingEnvironment(
             }
         return getDataLoader(loaderName) ?: throw NoDataLoaderFoundException("DataLoader with name $loaderName not found")
     }
+
+    fun isArgumentSet(path: String): Boolean {
+        val pathParts = path.split(".", "->").map { s -> s.trim() }
```

Comment:
Do we need to support 2 ways of specifying the path?

Reply:
I mentioned earlier that I just couldn't decide which one is nicer, but there's a reason to support both.
When just writing the path manually, the "bla->other" approach looks more concise, and is the nicer opinion. That's what I would personally use.
Some folks don't like strings that aren't compile time checked, and would prefer using constants. Codegen already generates those, so you could write something like:
```
isArgumentSet(DgsConstants.MUTATION.RECORD_INPUT_ARGUMENT.StartupEvent, DgsConstants.STARTUPEVENTINPUT.ApplicationName)
```

In that case the varargs are nicer because you don't need to do a string concat just to create the argument.

Reply:
Somewhat related; what I really wanted is some sort of method reference, but I don't think we can get that to work with Java. E.g. something like what you would do with a lambda:

```
isArgumentSet(personInput::getName)
```